app/auth/views.py

```diff
@@ -0,0 +1,14 @@
+from flask import render_template
+from . import auth
+
```

Comment:
It would be nice if you delete the line. We will use 2 lines after a function, method and after import statements to be consistent with pycodestyle `^2.11.1`. 
Look also lines below `login()`